### PR TITLE
add new redirects file for fishtown to getdbt.com/blog/

### PR DESCRIPTION
**Pull request by JKarlavige**
This adds Netlify redirects to redirect from blog.fishtownanalytics -> getdbt.com/blog/

## Preview Links
https://deploy-preview-4--blog-fishtownanalytics.netlify.app/blog

### CODE REVIEW

**Comment by lfracalossiFA** [comment]
Is this ready for review yet, @JKarlavige?

**Comment by JKarlavige** [comment]
> Is this ready for review yet, @JKarlavige?

Yes it's ready, add you and Krevitt for review

File: _redirects [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Redirects from blog.fishtownanalytics.com -> getdbt.com/blog/
```
Comment (lfracalossiFA) [inline _redirects:1]: A shame we can't use splat redirects for these! I assume the numbers at the end of the Ghost URLs prohibits that, yes?

File: _redirects [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+/its-time-for-open-source-analytics-194902ae5c5 https://getdbt.com/blog/it-s-time-for-open-source-analytics/
+
+/analyzing-your-mailchimp-data-part-1-measuring-core-email-performance-8c45640155f9 https://getdbt.com/blog/
+
+/analyzing-your-stripe-data-part-1-measuring-subscription-mrr-603d3597e289 https://getdbt.com/blog/modeling-subscription-revenue/
+
+/welcome-youre-now-a-software-developer/ https://getdbt.com/blog/welcome-youre-now-a-software-developer/
+
+# blog.fishtownanalytics.com -> getdbt.com/blog/ redirects
+/author/lauren/ https://www.getdbt.com/author/lauren-craigie/ 301!
```
Comment (lfracalossiFA) [inline _redirects:102]: I don't think the "301!" is necessary, since those files don't exist on this site, but no harm in keeping it.

**Comment by lfracalossiFA** [comment]
@JKarlavige I'm getting a 404 when I go to https://deploy-preview-4--blog-fishtownanalytics.netlify.app/. I'm assuming this is because getdbt.com/blog/ isn't live yet, correct?

**Comment by JKarlavige** [comment]
That's correct, as long as the redirects are firing it should be good. But i don't expect any of the pages it redirects to to exist until tomorrow after take-live

**Comment by lfracalossiFA** [comment]
Also, do we need to do anything with the redirects.json file in this directory? I'm not actually sure what it's hooked up to.

**Comment by lfracalossiFA** [comment]
FWIW, some of the blog posts appear to already be live. Is that intentional? For example, I tried `blog.getdbt.com/using-dbt-docs-fae6137da3c3` and it correctly redirected to `https://blog.getdbt.com/using-dbt-docs/`, which is already live.

I know they're not linked anywhere, so not a big deal, but just curious.

**Review by lfracalossiFA — APPROVED** [review]
I still don't entirely understand how all the pieces fit together here, but all the redirects are syntactically correct, and appear to work, so I think we're good to go.

File: _redirects (hunk 1 above)
Comment (JKarlavige) [reply _redirects:1]: Yeah I assume that prevents us from using splat, not too familiar with those urls or what they used to be though 😬

**Comment by JKarlavige** [comment]
Merging this in. Should have no effect until CNAME pointed to Netlify and domain alias setup.

**Pull request merged**